crates/driver/openapi.yml

```diff
@@ -379,6 +382,9 @@ components:
             type: object
             additionalProperties:
               $ref: "#/components/schemas/BigUint"
+          gas:
+            type: integer
+            description: How many units of gas this solution is estimated to cost.
```

Comment:
Do we really need this on the solve endpoint? What is the autopilot going to do with this information?